portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+window.onload = function() {
+  setTimeout(toggleContent, 3500);
+}
+
 /**
- * Adds a random greeting to the page.
+ * Generate a random fact about myself
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function randomFact() {
+  const facts = [
+    'I can play the guitar!', 
+    'I broke my thumb before...',
+    'My hair turned brown from swimming (chlorine).',
+    'I love singing.',
+    'I am a Hong Kong PR.'
+  ];
+  var fact = facts[Math.floor(Math.random() * facts.length)];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  document.querySelector('#fact').innerHTML = fact;
```

Comment:
since the fact doesnt contain any HTML tag, should we use innerText instead of innerHTML?

Reply:
Was intending to use HTML in the future, but will use innerText for now until I add in HTML